ref/gl/gl_sprite.c

```diff
@@ -200,22 +202,25 @@ void Mod_LoadSpriteModel( model_t *mod, const void *buffer, qboolean *loaded, ui
 
 	for( i = 0; i < mod->numframes; i++ )
 	{
-		frametype_t frametype = pframetype->type;
+		frametype_t frametype;
+
+		memcpy( &dframetype, pframetype, sizeof( dframetype ));
```

Comment:
Do you really need `dframetype` in outer scope? 

Maybe its scope could be lowered to this block?